### PR TITLE
Small ComboBox improvements

### PR DESCRIPTION
**Pull request by sebastianbenz**
* Fix content being pushed down
* some markup clean up
* use inline css icon instead of font
* show background for suggest box

### CODE REVIEW

**Comment by sebastianbenz** [comment]
@diogo-bastos small fixes to your samples, main problem was that content below the suggest box was being pushed down.

File: src/30_Advanced/ComboBox.html [hunk 1]
```diff
@@ -12,32 +12,29 @@
 <head>
   <title>ComboBox</title>
   <meta charset="utf-8">
-  <link rel="canonical" href="<%host%>/..\30_advanced/combobox/">
+  <link rel="canonical" href="https://ampbyexample.com/..\30_advanced/combobox/">
```
Comment (kul3r4) [inline src/30_Advanced/ComboBox.html]: why do we need to specify the production URL?

File: src/30_Advanced/ComboBox.html [hunk 2]
```diff
@@ -158,44 +140,38 @@
...
+      <div class="search-container">
+        <input name="products-dropdown" type="hidden" [value]="foodId">
+        <input id="query" name="query" autocomplete="off" placeholder="Select a product" type="text" class="search-box" on="input-debounced:AMP.setState({ query: event.value, inputText: event.value, autosuggest: event.value, comboExpanded: true}), autosuggest-list.show"
+          [value]="inputText || ''" value="">
+
+        <button on="tap:autosuggest-list.toggleVisibility, AMP.setState({ comboExpanded: !comboExpanded })" class="search-submit" type="button">
+          <i [class]="comboExpanded ? 'arrow-up' : 'arrow-down'" class="arrow-down"></i>
+        </button>
+      </div>
+        <amp-list class="autosuggest-box" single-item items="." layout="fixed-height" height="350" src="https://ampbyexample.com/json/food-options.json" id="autosuggest-list" hidden>
```
Comment (kul3r4) [inline src/30_Advanced/ComboBox.html]: can we use relative url in the `src`?

File: src/30_Advanced/ComboBox.html (hunk 1 above)
Comment (sebastianbenz) [reply src/30_Advanced/ComboBox.html]: good find. done

File: src/30_Advanced/ComboBox.html (hunk 2 above)
Comment (sebastianbenz) [reply src/30_Advanced/ComboBox.html]: yes. done

**Comment by sebastianbenz** [comment]
PTAL

**Comment by sebastianbenz** [comment]
...and it's live: https://ampbyexample.com/advanced/combobox/preview/

Thanks for the review.

**Pull request merged**